### ISSUE TITLE
Terminal Not closing with Delete icon

### ISSUE THREAD

**Issue by Myestery** [lglong519/vscode-extension#5]
Many times when I open a workspace, I get many terminal instances from terminal tools.
I try to exit them using the delete icon or even the exit command but it always pops back up immediately.
Is there any option to enable closing the terminal?

**Comment by Roma-Kyrnis** [comment]
Hi, I've got the same issue. I've disabled the terminal because I didn't find a solution. Do you have any?

**Comment by AimForNaN** [comment]
@lglong519 This issue is annoying enough to want to uninstall the extension.

**Comment by dreaded369** [comment]
yep - i disabled it. everytime vscode is opened it adds a terminal-tools window, and everyone one of them auto re-opens when closed.

**Comment by Roma-Kyrnis** [comment]
Thank you!

вт, 27 черв. 2023 р. о 16:37 dreadedhamish ***@***.***> пише:
